src/app/extensions/quoting/shared/quote-interactions/quote-interactions.component.html

```diff
@@ -6,8 +6,13 @@
         data-testing-id="submit-quote-request"
         type="submit"
         class="btn btn-primary float-right"
-        [disabled]="quote.items.length === 0"
-        (click)="submit()"
+        [ngClass]="{ disabled: quote.items.length === 0 }"
+        [attr.aria-disabled]="quote.items.length === 0"
+        [attr.aria-label]="
```

Comment:
Problems with aria-label:
Overwrites the button's visible text for screen readers
Creates inconsistency between what sighted users see vs. what screen reader users hear
The disabled state message may not be announced when the button becomes disabled

pls. check the following approach (and use it for all similar occurences):
```

<div 
  id="submit-button-help" 
  aria-live="polite" 
  class="sr-only"
  *ngIf="form.invalid && addressForm.submitted"
>
  {{ 'forms.required.button.help.disabled' | translate }}
</div>

<button
  type="submit"
  class="btn btn-primary"
  [disabled]="form.invalid && addressForm.submitted"
  [attr.aria-describedby]="(form.invalid && addressForm.submitted) ? 'submit-button-help' : null"
>
  {{ buttonLabel | translate }}
</button>

```